apps/portal/app/routes/app+/quest+/index.tsx

```diff
@@ -93,14 +100,11 @@ export default function Quests() {
     inviteCodes,
     mintCount,
     relicHoldCount,
+    relicPoints,
     protocolFees,
     details,
   } = useLoaderData<typeof loader>()
 
-  const nftMintPoints = mintCount * 2000000
```

Comment:
Is all of this math now fully on the BE endpoint?

Reply:
Yea, no math needed anymore since we are just deriving the points from the snapshots